pkg/kotsadm/version/kotsadm_version.go

```diff
@@ -73,11 +73,9 @@ func KotsadmPullSecret(namespace string, options types.KotsadmOptions) *corev1.S
 		return nil
 	}
 
-	secret, _ := registry.PullSecretForRegistries([]string{options.OverrideRegistry}, options.Username, options.Password, namespace, "")
-	if secret == nil {
-		return nil
-	}
+	secrets, _ := registry.PullSecretForRegistries([]string{options.OverrideRegistry}, options.Username, options.Password, namespace, "")
 
+	secret := &secrets.AdminConsoleSecret
```

Comment:
this could be nil, the check for that is removed

Reply:
that function always creates that secret.  that check was not necessary.  also that's no longer a pointer.